src/main/clm_varpar.F90

```diff
@@ -203,7 +203,8 @@ subroutine clm_varpar_init(actual_maxsoil_patches, surf_numpft, surf_numcft)
     max_patch_per_col= max(maxsoil_patches, surf_numcft, maxpatch_urb)
 
     nlevsoifl   =  10
-    nlevurb     =  5
+    ! X. Li: changed below from 5 to 10 to match new surface data
```

Comment:
- [ ] Remove this new comment and code.  This will be taken care of when the ctsm5.2_mksurfdata branch is merged in.